src/anemoi/models/models/encoder_processor_decoder.py

```diff
@@ -58,48 +63,50 @@ def __init__(
 
         self._define_tensor_sizes(config)
 
-        # Create trainable tensors
-        self._create_trainable_attributes()
-
         # Register lat/lon
-        self._register_latlon("data", self._graph_name_data)
-        self._register_latlon("hidden", self._graph_name_hidden)
+        for name in self._graph_mesh_names:
```

Comment:
This could just be `in self._graph_data` for simplicity. `name` should probably be renamed into something like `mesh_key` or `grid_key`.

Reply:
and mesh_name? do you think is it a good option?